### ISSUE TITLE
Bug in imports from benchmark

### ISSUE THREAD

**Issue by danielandresarcones** [TNO/harlow#12]
The [samplers benchmark](https://github.com/TNO/harlow/blob/master/tests/benchmark_tests/benchmark_samplers.py) seems to have some bugs in the imports. Specifically lines 18 and 19:

```
from harlow.sampling.probabilistic_sampling import probabilistic_sampler
from harlow.sampling.random_sampling import latin_hypercube_sampler
```
should be
```
from harlow.sampling.probabilistic_sampling import ProbabilisticSampler
from harlow.sampling.random_sampling import LatinHypercube
```

I had an additional problem with the import in line 22, with error "Cannot import test.integration_tests.test_functions. No module named tests.integration_tests.test_functions".
```
from tests.integration_tests.test_functions import hartmann, peaks_2d, stybtang
```
It can be my own python paths setup, but I solved it adding before the imports
```
import inspect

currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
parentdir = os.path.dirname(os.path.dirname(currentdir))
sys.path.insert(0, parentdir)
```
Also, I would delete line 26, it is redundant with line 25.
```
stop_thresh = 0.01  # For RMSE or 0.005 - 0.0025
stop_thresh = None
```

**Comment by JanKoune** [comment]
Hi Daniel,

Thank you for the detailed issue! The import bugs should be fixed by PR #13. I should also mention that it is no longer necessary to specify the full "path" to the sampling and surrogating functions in your imports. Instead all surrogates and samplers can be imported from `harlow.surrogating` and `harlow.sampling` respectively. As an example:

```Python
from harlow.sampling.probabilistic_sampling import ProbabilisticSampler
```
can be replaced with:
```Python
from harlow.sampling import ProbabilisticSampler
```

Regarding your second point (import in line 22), I could not recreate the issue. I suspect that this is related to your particular setup like you mentioned. This should be fixed by #14.

**Comment by JanKoune** [comment]
Closing the issue after merging #14. Please re-open if you continue getting the `No module named tests.integration_tests.test_functions` error.
